### ISSUE TITLE
`samples.plot.kde_2d()` should show the documentation from `kde_contour_plot_2d`

### ISSUE THREAD

**Issue by lukashergt** [williamjameshandley/anesthetic#220]
Currently the docstring of `samples.plot.kde_2d()` reads as
```python
"""KDE plot: See anesthetic.plot.kde_contour_plot_2d."""
```

I think this is not good enough. For `samples.plot_2d()` it is difficult to list all the possible kwargs of the various different plot kinds, but we need some easy accessible way to check for the possible kwargs for `kde_2d`. My standard approach would be to write `samples.kde_2d(` and then press `Shift+Tab` to get function signature and docstring.

**Comment by williamjameshandley** [comment]
Agreed -- we could use this as a test-bed for documentation re-use, which should still show up in your autocompletion.
